valhalla/baldr/graphtile.h

```diff
@@ -594,9 +594,9 @@ class GraphTile {
   inline bool IsClosedDueToTraffic(const GraphId& edge_id) const {
     if (!traffic_tile())
       return false;
-    auto live_speed = traffic_tile.getTrafficForDirectedEdge(edge_id.id());
+    auto& live_speed = traffic_tile.getTrafficForDirectedEdge(edge_id.id());
```

Comment:
technically this `auto` includes `volatile` right? nitpick but it wouldnt hurt to put it just for understandings sake or easy readabiltiy